grails-app/views/occurrence/_recordCore.gsp

```diff
@@ -1,891 +1,1075 @@
 <%@ page import="org.apache.commons.lang.StringUtils" %>
+
 %{--<% Map fieldsMap = new HashMap(); pageContext.setAttribute("fieldsMap", fieldsMap); %>--}%
 <%-- g:set target="${fieldsMap}" property="aKey" value="value for a key" /--%>
 <g:set var="fieldsMap" value="${[:]}"/>
+
 <div id="occurrenceDataset">
-<g:render template="sandboxUploadSourceLinks" model="[dataResourceUid: record?.raw?.attribution?.dataResourceUid]" />
-<h3><g:message code="recordcore.occurencedataset.title" default="Dataset"/></h3>
-<table class="occurrenceTable table table-sm table-bordered table-striped table-condensed" id="datasetTable">
-<!-- Data Provider -->
-    <alatag:occurrenceTableRow annotate="false" section="dataset" fieldCode="dataProvider" fieldName="Data provider">
-        <g:if test="${record.processed.attribution.dataProviderUid && collectionsWebappContext}">
-            ${fieldsMap.put("dataProviderUid", true)}
-            ${fieldsMap.put("dataProviderName", true)}
-            <a href="${collectionsWebappContext}/public/show/${record.processed.attribution.dataProviderUid}">
+    <g:render template="sandboxUploadSourceLinks" model="[dataResourceUid: record?.raw?.attribution?.dataResourceUid]" />
+
+    <h1><g:message code="recordcore.occurencedataset.title" default="Dataset"/></h1>
+
+    <table class="occurrenceTable table table-sm table-bordered table-striped table-condensed" id="datasetTable">
```

Comment:
table-condensed was renamed to table-sm in Bootstrap 4. Should remove when stumbling on them.